### PR TITLE
fix getting signatures in jedi 0.17

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

**Comment by Carreau** [comment]
@davidhalter does that looks reasonable to you ? Or is there a cleaner way ?

**Comment by davidhalter** [comment]
Looks reasonable, but I would handle the case of `get_signatures() == []` as well. This makes it easier in the future, so you don't have to depend on type (which was a bad idea in the first place - my fault).

File: IPython/core/completer.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 Syntax like magics.
 
 Latex and Unicode completion
-============================
+====================:========
```
Comment (davidhalter) [inline IPython/core/completer.py:12]: Probably not intentional :)

Otherwise, LGTM.

Comment (Carreau) [reply IPython/core/completer.py]: Doh.. force-pushed.

**Comment by Carreau** [comment]
actually no need to backport, 7.x still use `completions()` and not `complete()`

**Pull request merged**